upup/models/config/components/kube-apiserver/kube-apiserver.options

```diff
@@ -16,3 +16,7 @@ KubeAPIServer:
   LogLevel: 2
   AllowPrivileged: true
   Image: {{ Image "kube-apiserver" }}
+  KubeletPreferredAddressTypes:
```

Comment:
I've realized you can only pass this in 1.5 or later.

I think you need to put this in a `_k8s_1_5` magic directory (like we do for ConfigureCBR0, though the "other way round").  Or - if these are the defaults - simply not specify them unless you are overriding.

Are we changing the defaults here?